### PR TITLE
cryptography 43.0.3

### PR DESCRIPTION
**Pull request by boldorider4**
cryptography 43.0.1 

**Destination channel:** Defaults

### Links

- [PKG-5800]
- dev_url:        https://github.com/pyca/cryptography/tree/43.0.3
- conda_forge:    https://github.com/conda-forge/cryptography-feedstock/blob/main/recipe/meta.yaml
- pypi:           https://pypi.org/project/cryptography/43.0.3
- pypi inspector: https://inspector.pypi.io/project/cryptography/43.0.3

### Explanation of changes:

- new version number

[PKG-5800]: https://anaconda.atlassian.net/browse/PKG-5800?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lorepirri** [comment]
could you please update the description with the correct version and also the ticket,

this depends on https://github.com/AnacondaRecipes/cryptography-vectors-feedstock/pull/22 and the two versions must be kept in sync

**Comment by danpetry** [comment]
Where's the dependency between this and cryptography-vectors expressed? It's not in the dep list in the feedstock.

Also, I'm assuming this is needed - there's a comment that it might not be needed, in the jira ticket. I guess it's good to keep it updated anyway.

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,7 +49,6 @@ test:
     - pip
     - pytest >=6.2.0
     - pytest-benchmark
```
Comment (lorepirri) [inline recipe/meta.yaml:51]:
```suggestion
```

**Review by lorepirri — APPROVED** [review]

**Review by danpetry — APPROVED** [review]

**Comment by danpetry** [comment]
assuming this just needs cryptography-vectors to be uploaded from zeus

**Pull request merged**